### ISSUE TITLE
Migrate all relevant post data to `posts.json` and convert `Posts.jsx` to app-side processing only

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#1156]
Just spent a while debugging my post not showing up because I was editing the wrong contents file (should be Posts.jsx now)

**Comment by anth-volk** [comment]
Reverted by #1200. Reopening.

**Comment by dotslashbit** [comment]
@anth-volk Can you clarify, if this issue still exists on not. If yes, then please give some more details on this issue?

**Comment by anth-volk** [comment]
@dotslashbit This issue does still exist.

Currently, we have two files to store news articles: `posts.json`, which is a simple JSON object, and `posts.jsx`, a file that contains the articles in a standard JavaScript object, as well as some functions used for processing this data. However, at the moment, we have a few Python files that read the posts, and they require the JSON object, since they of course can't interact with JavaScript code. A previous contribution erroneously removed the `posts.json` file, which then broke the Python files, and we didn't notice this until deployment because our tests don't touch the Python, and the Python's only used on deploy.

So basically, we probably should update this issue to be something like, remove the posts from `posts.jsx`, add the newer posts we have to `posts.json`, import those into the JSX file if necessary, and update various import references, etc. - at least that's what I'm thinking at the moment.
